app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ActiveRecord::Base
   has_secure_password
 
   enum role: %w(company admin)
+
```

Comment:
We decided that we do want a super admin roll. Do you want to add that into this PR or a separate one?